stock_reorder_forecast/models/procurement_order.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    #disable making PO's from procurement orders
+
+    @api.model
+    def _run(self, procurement):
+        if procurement.rule_id and procurement.rule_id.action == 'buy':
+            return  super(ProcurementOrder, self.with_context(
+                skip_po=True))._run(procurement)
+        return super(ProcurementOrder, self)._run(procurement)
+
+
+    @api.multi
+    def make_po(self):
+        for procurement in self:
+            if self.context['skip_po'] == True:
+                return []
```

Comment:
read https://github.com/OCA/OCB/blob/9.0/addons/procurement/procurement.py#L195 and think again if this is a good idea